languages/elixir/exercises/concept/exceptions/test/test_helper.exs

```diff
@@ -0,0 +1,7 @@
+options = []
+options = if Code.ensure_compiled(RPNCalculator.Exception.DivisionByZeroError) == {:error, :nofile}, do: [undefined_division_by_zero_error_module: true] ++ options, else: options
+options = if Code.ensure_compiled(RPNCalculator.Exception.StackUnderflowError) == {:error, :nofile}, do: [undefined_stack_underflow_error_module: true] ++ options, else: options
+options = if not function_exported?(RPNCalculator.Exception.DivisionByZeroError, :__struct__, 0), do: [undefined_struct_division_by_zero_error_exception: true] ++ options, else: options
+options = if not function_exported?(RPNCalculator.Exception.StackUnderflowError, :__struct__, 0), do: [undefined_struct_stack_underflow_error_exception: true] ++ options, else: options
```

Comment:
I think this would be a bit simpler to read, and should work exactly the same:

```suggestion
options = [
  undefined_division_by_zero_error_module:
    Code.ensure_compiled(RPNCalculator.Exception.DivisionByZeroError) == {:error, :nofile},
  undefined_stack_underflow_error_module:
    Code.ensure_compiled(RPNCalculator.Exception.StackUnderflowError) == {:error, :nofile},
  undefined_struct_division_by_zero_error_exception:
    not function_exported?(RPNCalculator.Exception.DivisionByZeroError, :__struct__, 0),
  undefined_struct_stack_underflow_error_exception:
    not function_exported?(RPNCalculator.Exception.StackUnderflowError, :__struct__, 0)
]
```

Reply:
Sure, I was trying to avoid keys in the kw list, but this looks nicer given the long arguments.